source/types/options.ts

```diff
@@ -71,7 +71,7 @@ export interface Options extends Omit<RequestInit, 'headers'> {
 
 	Accepts any plain object or value, which will be `JSON.stringify()`'d and sent in the body with the correct header set.
 	*/
-	json?: unknown;
+	json?: Record<string, unknown>;
```

Comment:
Correct me if it can be not only an object

Reply:
JSON can have top-level array or primitive value.

Reply:
So would it be ok to have `PrimitiveType | Record<string, unknown>`?

```ts
type PrimitiveType = number | string | boolean | undefined | null;
```

Reply:
Btw, what is the expected value after calling `.json()` if you pass primitives?

Here in test case I see that it's string instead of `boolean`:

```ts
test('POST JSON with falsey value', async t => {
	// #222
	const server = await createHttpTestServer({bodyParser: false});
	server.post('/', async (request, response) => {
		response.json(await parseRawBody(request));
	});

	const json = false;
	const responseJson = await ky.post(server.url, {json}).json();

	t.deepEqual(responseJson, json.toString());

	await server.close();
});
```

Can you please describe the rules so I will be on the same page?

Reply:
> So would it be ok to have Primitive | Record<string, unknown>?

No, it used to be that, but it caused problems with `interface`s. `git blame` it and you'll find the relevant thread.